sources/packages/backend/apps/api/src/route-controllers/application/application.controller.service.ts

```diff
@@ -290,6 +294,57 @@ export class ApplicationControllerService {
         applicationOfferingChangeRequest?.applicationOfferingChangeRequestStatus,
       hasBlockFundingFeedbackError,
       eCertFailedValidations: [...eCertValidationResult.failedValidations],
+      changeRequestInProgress,
+    };
+  }
+
+  /**
+   * Gets the information of an in-progress change request for the application, if one exists.
+   * Checks for he existence of an in-progress change request for the application to get a list
+   * of sub-processes status, for instance, income verifications and supporting user information.
+   * @param applicationId current application ID to verify a possible in progress change request.
+   * @param options options.
+   * - `studentId` student ID used for authorization, when required.
+   * @returns information of the in-progress change request for the application, if one exists.
+   */
+  private async getInProgressChangeRequestDetails(
+    applicationId: number,
+    options?: { studentId?: number },
+  ): Promise<ChangeRequestInProgressAPIOutDTO | undefined> {
+    const [inProgressChangeRequest] =
+      await this.applicationService.getApplicationsVersionByEditStatus(
+        applicationId,
+        APPLICATION_EDIT_STATUS_IN_PROGRESS_VALUES,
+        {
+          studentId: options?.studentId,
+        },
+      );
+    if (!inProgressChangeRequest) {
+      return undefined;
```

Comment:
will `return;` have the same effect?

Reply:
Yes, it was explicit returned make the intention clear and aligned with the explicit return types.